### ISSUE TITLE
extend-immutable-calls: how to configure?

### ISSUE THREAD

**Issue by DavidVujic** [PyCQA/flake8-bugbear#212]
I haven't been able to make the [recently released feature](https://github.com/PyCQA/flake8-bugbear/pull/204) `extend-immutable-calls` to work. 

Have I configured it wrong? 

I have verified that `flake8` recognizes the config, by setting the line-length to 10 and verified flake8 warnings about line length.

```bash
flake8 --version

4.0.1 (flake8-bugbear: 21.11.29, mccabe: 0.6.1, pycodestyle: 2.8.0, pyflakes: 2.4.0) CPython 3.10.0 on
Darwin
```

`example.py`
```python
import fastapi


def example_fn(data: str = fastapi.Header(None)):
    pass
```

`setup.cfg`
```
[flake8]
max-line-length = 120
extend-immutable-calls = ["fastapi.Header"]
```

```bash
flake8 --show-source

./example.py:4:28: B008 Do not perform function calls in argument defaults.  The call is performed only once at function definition time. All calls to your function will reuse the result of that definition-time function call.  If this is intended, assign the function call to a module-level variable and use that variable as a default value.
def example_fn(data: str = fastapi.Header(None)):
                           ^
```

**Comment by aberres** [comment]
Same problem here - I tried to silence `pathlib.Path`.

**Comment by sco1** [comment]
You want:

```
[flake8]
max-line-length = 120
extend-immutable-calls = fastapi.Header
```

**Comment by cooperlees** [comment]
Do we feel we should add anything more to the README here? I'd accept a PR if so. Otherwise do we close?

**Comment by sco1** [comment]
Even though it's just `ConfigParser` doing it's thing, it wouldn't hurt to add an example to the README (see also: #215).

I'm happy to include in #218 if you feel it's appropriate.

**Comment by aberres** [comment]
Ah, great thanks.

And the matching happens by name, right?

So in the following case I would need `extend-immutable-calls = Path` to silence the warning.

```
from pathlib import Path

def foo(bar=Path()):
   ...
```

**Comment by aberres** [comment]
And as we are here already: Would it make sense to add a hint to this parameter to the error message of B008?

**Comment by DavidVujic** [comment]
Great, thank you for clarifying! 

I had thought that the values should be defined as a (Python) array. Now, when typing something like this (for more than one fastAPI thing to allow), everything works as expected:
```
extend-immutable-calls = fastapi.Header, fastapi.Depends
```

**Comment by cooperlees** [comment]
Please at least add to the README to make it clearer. The message, I’d you can make a sane suggestion in code please do too. Thanks!

**Comment by copdips** [comment]
> Great, thank you for clarifying!
> 
> I had thought that the values should be defined as a (Python) array. Now, when typing something like this (for more than one fastAPI thing to allow), everything works as expected:
> 
> ```
> extend-immutable-calls = fastapi.Header, fastapi.Depends
> ```

You can use list format like this:

```yaml
extend-immutable-calls =
    fastapi.Header,
    fastapi.Depends,
```

**Comment by npip99** [comment]
If anyone is using `pyproject.toml`,

```
[tool.ruff.lint.flake8-bugbear]
extend-immutable-calls = [
    # Whitelisted default arguments
    "fastapi.Header",
    "fastapi.Depends",
]
```
